mythril/laser/ethereum/instructions.py

```diff
@@ -1728,7 +1773,7 @@ def call_post(self, global_state: GlobalState) -> List[GlobalState]:
         instr = global_state.get_current_instruction()
 
         try:
-            callee_address, _, _, value, _, _, memory_out_offset, memory_out_size = get_call_parameters(
+            callee_address, callee_account, call_data, value, gas, memory_out_offset, memory_out_size = get_call_parameters(
```

Comment:
why did you add this change?

Reply:
there was an extra `_` there compared to the return statement of the get_call_parameters. Probably we shouldn't return too many parameters :sweat_smile:, like returning a struct might be good.

Reply:
Yeah, `get_call_parameters` returns 7 parameters, and 8 were expected, so a `ValueError` was being raised and caught every time.